### PR TITLE
feat: add visualization quick actions

### LINKED ISSUE #15: Add Dashboard quick action to display moodle analytics

### PR DESCRIPTION
**Pull request by pyphilia**
close #15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,11 +4,13 @@
...
   "author": "HEAI-FR",
   "contributors": [
-    "Uchendu Nwachukwu"
+    "Uchendu Nwachukwu",
+    "Kim Lan Phan Hoang"
   ],
   "homepage": ".",
   "license": "MIT",
   "dependencies": {
+    "@graasp/moodle": "link:../react-app/graasp-app-moodle-analytics",
```
Comment (pyphilia) [inline package.json:13]: This should change once the package is published on npm.

Comment (juancarlosfarah) [reply package.json:13]: Can you try now?

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Very nice! Just a minor change to import from NPM.

**Comment by pyphilia** [comment]
It is working now (I had some issues with the build version of the code...)

**Review by juancarlosfarah — APPROVED** [review]

**Pull request merged**